python/cugraph/bfs/test_bfs.py

```diff
@@ -74,10 +74,10 @@ def base_call(M, start_vertex):
     return vertex, dist
 
 
-DATASETS = ['/datasets/networks/dolphins.mtx',
-            '/datasets/networks/karate.mtx',
-            '/datasets/networks/polbooks.mtx',
-            '/datasets/golden_data/graphs/dblp.mtx']
+DATASETS = ['../datasets/dolphins.mtx',
+            '../datasets/karate.mtx',
+            '../datasets/polbooks.mtx',
+            '../datasets/netscience.mtx']
```

Comment:
So this works if we run pytest under python, but this will not work if we run this under python/cugraph or python/cugraph/bfs. I sometimes do this to run just a subset of tests. This is my biggest concern on using a relative path. I thought about using an environmental variable (e.g. something like RAPIDS_CUGRAPH_DATASET_PATH) to override the default path for such cases.

Reply:
That correct but this is a separate issue.
@rlratzel is currently looking into adding paths as a parameter.

Reply:
In the meantime, you can still run one particular test with the pystest command from the main python dir.

Reply:
Yeah it seems Jenkins run it from another directory as well :( 
I need to fix that

Reply:
Fixed after 0d7f1d5
@rlratzel is currently working on a better solution that allows datasets to be anywhere by looking at an environment variable.